api/proto/agent/health.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+package health;
```

Comment:
in other .proto files, we used the "package name" = service (scheduler, controller, agent) name convention

Since I don't understand this interface, I can't say which is best, but the naming convention already in place allowed me to understand which .proto file was to be used by which other service - Here, I can't infer this information from the name of this file or of its package

Reply:
This is horrible to do that because the if you have multiple protos the name will be the same. Therefore the codegen will be lost if you need to build multiple protos

Reply:
> This is horrible to do that because the if you have multiple protos the name will be the same. Therefore the codegen will be lost if you need to build multiple protos

Fair enough for the file name, but we need a consistent naming convention
However about the package name, shouldn't it be agent instead of health though?

Reply:
No because the package name is the titlte for what is inside the package. Like if you are making an interface for coffee exchange you could name the package coffee but not nespresso

Reply:
I'm unsure
I was looking at https://protobuf.dev/programming-guides/style/#packages which says
> Package names should be in lowercase. Package names should have unique names based on the project name, and possibly based on the path of the file containing the protocol buffer type definitions.

It's really broad so i guess any name is right as long as it's our convention

But then, do we need to put https://github.com/dev-sys-do/sealci/blob/main/api/proto/scheduler/agent.proto in the health package too ? Since all its messages are about an agent's health